### PR TITLE
[cloud-provider-yandex]  Change default root disk size for master and cloud permanent nodes to 50 GB

### PR DESCRIPTION
**Pull request by name212**
## Description
Change defaults in terraform.
Add migration for adding old default disk size (20 GB) to provider cluster configuration to prevent terraform state changing.
Remove diskSizeGB from e2e tests.

## Why do we need it, and what problem does it solve?
Difference between documentation and real value in cluster.

## Checklist
- [x] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: cloud-provider-yandex
type: fix
summary: Change default root disk size for master and cloud permanent nodes to 50 GB
impact_level: default
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @name212. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by name212** [comment]
/e2e/abort yandex-cloud;WithoutNAT;containerd;1.25 9348541159 9348541159-1-con-1-25 /sys/deckhouse-oss/install:pr8421 redos@178.154.225.37

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @name212. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @name212. I've started the workflow for label 'e2e/run/yandex-cloud'...

File: modules/030-cloud-provider-yandex/hooks/migrate_disk_gb_size_before_changing_default.go [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+	}
+
+	if version, ok := cm.Data["version"]; ok {
+		return version, nil
+	}
+
+	return "", nil
+}
+
+func createFirstDeschedulerCR(input *go_hook.HookInput) error {
```
Comment (RomanenkoDenys) [inline modules/030-cloud-provider-yandex/hooks/migrate_disk_gb_size_before_changing_default.go:104]: why so strange name ?

**Review by RomanenkoDenys — COMMENTED** [review]
strange naming

File: modules/030-cloud-provider-yandex/hooks/migrate_disk_gb_size_before_changing_default.go (hunk 1 above)
Comment (name212) [reply modules/030-cloud-provider-yandex/hooks/migrate_disk_gb_size_before_changing_default.go]: fixed

**Review by RomanenkoDenys — APPROVED** [review]
LGTM

**Pull request merged**